content/react/concepts/state/state.md

```diff
@@ -13,43 +13,48 @@ CatalogContent:
   - 'paths/front-end-engineer-career-path'
 ---
 
-The `state` object is where we store data in a component that is expected to change over time. When the `state` object changes, the component re-renders.
+The **`state`** object holds data in a [component](https://www.codecademy.com/resources/docs/react/components) that is expected to change over time. It is created and maintained by the component itself, and when it changes, the component re-renders.
 
 [Props](https://www.codecademy.com/resources/docs/react/props) are passed down by parent components, whereas state is created and maintained by the component itself.
 
 Class and functional components handle state differently. Functional components use [hooks](https://www.codecademy.com/resources/docs/react/hooks) to manage state. The following will address how class components manage state.
 
-## Creating the `state` Object
+## Syntax
 
-The `state` object is initialized in the component's `constructor()`:
-
-```js
-class Car extends React.Component {
+```pseudo
+class ComponentName extends React.Component {
+  /* With constructor */
   constructor(props) {
     super(props);
-    // The state object
     this.state = {
-      brand: 'Chevrolet',
-      model: 'Malibu',
-      color: 'white',
-      year: 1998,
+      property: value,
```

Comment:
Can we include manual binding here? It's a bit confusing because this pseudo code makes it seem as if we would use both the property initializer and constructor-declared-state within one class, when this is not the case (it's one or the other it seems). Maybe we can make this clear in an explanation below and include manual binding with an explanation as well.

Reply:
Alternatively, because I do see some of this explained below, we can create two example classes here. One pseudo code block with manual binding, and one without.

Reply:
My vote is for an explanation below the \```pseudo block.

Reply:
💯

Reply:
Ah ok, I see. Thanks!